apis/nucleus/src/index.js

```diff
@@ -346,8 +346,9 @@ function nuked(configuration = {}) {
            * @param {string=} options.title Custom title, defaults to fieldname
            * @param {string=} [options.direction=ltr] Direction setting ltr|rtl.
            * @param {string=} [options.listLayout=vertical] Layout direction vertical|horizontal
-           * @param {boolean=} [options.search=true] To show the search bar
-           * @param {boolean=} [options.toolbar=true] To show the toolbar
+           * @param {boolean=} [options.search=true] Show the search bar
+           * @param {boolean=} [options.toolbar=true] Show the toolbar
+           * @param {boolean=} [options.checkboxes=false] Show values as checkboxes instead of as fields
```

Comment:
I don't see any rangeSelect option. Is it missing?

Reply:
✅